src/hotspot/share/gc/shared/gcBehaviours.hpp

```diff
@@ -28,14 +28,14 @@
 #include "memory/iterator.hpp"
 #include "oops/oopsHierarchy.hpp"
 
-// This is the behaviour for checking if a CompiledMethod is unloading
+// This is the behaviour for checking if a nmethod is unloading
```

Comment:
Maybe this should be *an* nmethod?

Reply:
Quote: "an" goes before words that begin with vowels.

Reply:
I don't think that holds if the 'n' is pronounced the way nmethod is pronounced.

Reply:
`grep` shows that we have both cases but `an nmethod` is used more.

Reply:
I will fix it here as you suggested but I am not touching other places.